packages/db/index.js

```diff
@@ -22,6 +22,7 @@ const uploadQuery = `
         created:inserted_at,
         updated:updated_at,
         backupUrls:backup_urls,
+        backup( url ),
```

Comment:
I believe you should run `https://github.com/web3-storage/web3.storage/blob/main/packages/db/package.json#L13C5-L13C63` to update the types? but I do not remember anymore how this used to work

Reply:
Thanks for suggesting this. I wouldn't have known. I ran that script and it produced this diff https://github.com/web3-storage/web3.storage/pull/2352/commits/df36cfbc116c89fa7988da50991c08a7d7a08d08